packages/shared/styles/components/SfModal.scss

```diff
@@ -18,11 +19,21 @@ $modal-border: solid 1px rgba($c-black, 0.5) !default;
     align-content: space-between;
     box-sizing: border-box;
     background-color: $modal-background-color;
-    width: $modal-width;
     max-width: 100%;
-    height: $modal-height;
-    padding: $modal-padding;
-    border: $modal-border;
+    height: 100vh;
+    width: 100vw;
+    padding: $modal-mobile-padding;
+    @media (min-width: $tablet-min) {
```

Comment:
We are not using media queries tablet widths anymore - can you please change that to desktop/mobile? :)